src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import { filterTodos } from '../utils/filterTodos';
+import { Todo } from '../types/Todo';
+
+type TodoFooterProps = {
+  todos: Todo[];
+  setVisibleTodos: (filteredList: Todo[]) => void;
+};
+
+export const TodoFooter: React.FC<TodoFooterProps> = ({
+  todos,
+  setVisibleTodos,
+}) => {
+  const [linkIsActive, setLinkIsActive] = useState({
+    all: true,
+    active: false,
+    completed: false,
+  });
+
+  return (
+    /* Hide the footer if there are no todos */
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todos.filter(todo => todo.completed === false).length} items left`}
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${linkIsActive.all ? 'selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={() => {
+            const filteredList = filterTodos(todos, 'All');
+
+            setVisibleTodos(filteredList);
+          }}
+          onFocus={() => {
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method